### PR TITLE
Fixes for apple M1

### PR DESCRIPTION
**Pull request by martenbohlin**
…le m1 silicone, since it can run under rosetta.

### CODE REVIEW

File: roles/install_kubectl_versions/tasks/process-install.yaml [hunk 1]
```diff
@@ -15,25 +15,62 @@
...
+        fail:
+          msg: "Could not find file and no fallback set."
+        when: go_arch_fallback is not defined
+      - name: Fetech kubectl checksum ({{ version }} fallback arch)
+        uri:
+          return_content: yes
+          url: https://dl.k8s.io/v{{ version }}/bin/{{ go_platform }}/{{ go_arch_fallback }}/kubectl.sha256
+        register: _kubectl_checksum
+        tags:
+          - avinode_devenv_install_kubectl
```
Comment (pedrohdz) [inline roles/install_kubectl_versions/tasks/process-install.yaml:38]: I'm wondering if we could use something like this:
- https://medium.com/opsops/how-to-break-from-the-loop-in-ansible-1e8ebb92be0d

**Review by pedrohdz — COMMENTED** [review]
Tests might be a good idea...  Let's talk...

File: roles/install_kubectl_versions/tasks/process-install.yaml [hunk 2]
```diff
@@ -14,26 +14,78 @@
...
-- name: Fetech kubectl checksum ({{ version }})
-  uri:
-    return_content: yes
-    url: https://dl.k8s.io/v{{ version }}/bin/{{ go_platform }}/{{ go_arch }}/kubectl.sha256
-  register: _kubectl_checksum
-  tags:
-    - avinode_devenv_install_kubectl
+- name: Install kubectl version
+  block:
+  - name: Create temp file for checksum
```
Comment (pedrohdz) [inline roles/install_kubectl_versions/tasks/process-install.yaml:19]: Indent?  Tabbing is inconsistent.

**Review by pedrohdz — COMMENTED** [review]
Let's discuss...

File: roles/install_kubectl_versions/tasks/process-install.yaml (hunk 2 above)
Comment (martenbohlin) [reply roles/install_kubectl_versions/tasks/process-install.yaml:19]: I can only se spaces and no tab characters. What is the inconsistency?

**Review by pedrohdz — APPROVED** [review]

**Pull request merged**